scripts/mobile-flow.js

```diff
@@ -69,8 +69,7 @@ module.exports = function mobileFlow(runner, argv, clientId) {
                   Promise.resolve(doSyncRecordsResult.clientRecs)
                 ])))
           .spread((syncResponse, createdRecord, clientRecs) => Promise.all([
-            // TODO: sync with acknowledgements (still old dataset hash)
-            Promise.resolve(syncResponse),
+            doSync(`${baseUrl}/mbaas/sync/result`, makeSyncBody('result', clientId, syncResponse.hash, queryParams(user.id).result, null, _.values(_.get(syncResponse, 'updates.applied')))),
```

Comment:
the `doSync` calls here have become pretty complex, would be nice to have a little function to clean up and DRY the invocation.